src/ol_infrastructure/substructure/keycloak/__main__.py

```diff
@@ -457,3 +457,26 @@
         ),
     )
     # Okta TEST SAML [END]
+
+if stack_info.env_suffix in ["qa", "rc"]:
```

Comment:
U mean qa and ci?

Reply:
No, I just want this to be in QA/RC.  I thought I saw somewhere that the value of `env_suffix` could be "qa" or "rc".

Reply:
In that case, then it just needs to be qa